chore(docs): Migrate the callouts to bullets (#3580)
kroxylicious/kroxylicious: kroxylicious-docs/docs/_modules/configuring/ref-configuring-proxy-example.adoc

        template: "$(topicName)"
defaultFilters:
  - encryption
virtualClusters:
  - name: my-cluster-proxy
    targetCluster:
      bootstrapServers: my-cluster-kafka-bootstrap.kafka.svc.cluster.local:9093
      bootstrapServerSelection:
        strategy: "round-robin"
      tls:
        trust:
          storeFile: /opt/proxy/trust/ca.p12
          storePassword:
            passwordFile: /opt/proxy/trust/ca.password
    gateways:
    - name: mygateway
      sniHostIdentifiesNode:
        bootstrapAddress: my-cluster-proxy.kafka:9092
        advertisedBrokerAddressPattern: broker$(nodeId).my-cluster-proxy.kafka
      tls:
        key:
          storeFile: /opt/proxy/server/key-material/keystore.p12
          storePassword:
            passwordFile: /opt/proxy/server/keystore-password/storePassword
----
where:

* `filterDefinitions` provides a list of named filter configurations.
** `name` identifies the filter
** `type` provides the type of filter, which is the Record Encryption filter using Vault as the KMS in this example.
** `config` The configuration specific to the type of filter.

> *** `tls` If required, you can provide TLS configuration used to establish the connection to the KMS, with key names under which TLS certificates are stored.

Reviewer: ```suggestion
* `tls` specifies the TLS configuration used to establish a connection to the KMS, including the key names under which TLS certificates are stored.
```

* `virtualClusters` specifies the virtual cluster defintions.
** `name` The name of the virtual cluster.
** `targetCluster` defines the physical upstream cluster.

> *** `bootstrapServers` The bootstrap addresses of the Kafka Cluster being proxied.

Reviewer: ```suggestion
*** `bootstrapServers` defines the bootstrap addresses of the Kafka Cluster being proxied.
```

*** `strategy` defines the strategy used for selecting an address from 'bootstrapServers' when the proxy receives a client connection. Options are 'round-robin' (default) or 'random'.
*** `tls` provides TLS configuration for the connection to the target cluster.
* `gateways` defines the gateway(s) of the virtual cluster defining how the virtual cluster is presented to the network.

> ** `sniHostIdentifiesNode` The identification scheme used to route traffic to brokers, which can be `sniHostIdentifiesNode` or `portIdentifiesNode`.

Reviewer: ```suggestion
** The gateway routing scheme is configured using either `sniHostIdentifiesNode` or `portIdentifiesNode`.
```
